### DISCUSSION TITLE
Revisit Our Naming Conventions for Volunteer Roles

### DISCUSSION THREAD

**Discussion by meg-gutshall** [Virtual-Coffee/VC-Community-Docs#352] [Documentation]
# Summary

As VC has changed and grown over the years, we've come to realize that some of the titles we've assigned to our volunteer roles now seem ambiguous – namely, **lead**, **leader**, **host**, and **coordinator**. Let's examine these roles and brainstorm which terms are appropriate to use in which situation, or if there are better alternatives available to us.

> [!NOTE]
> This is a continuation of a discussion that started in PR #350.

## Term Investigation

Below are terms I've found in the repo to describe various roles/teams and their sources.

### VCHI

- Mentors
- Maintainers, VCHI Maintainers
- Contributors
- VC Maintainers
- MC Team Leads

([source](https://github.com/Virtual-Coffee/VC-Contributors/blob/ad1eeb933d68f37cc47647c3f28aa75cd61753f0/VCHI/Roles%20and%20things.md))

### Coffee Table Groups

- Leader(s) ([source](https://github.com/Virtual-Coffee/VC-Contributors/blob/ad1eeb933d68f37cc47647c3f28aa75cd61753f0/coffee-table-groups/coffee-table-groups.md)), Coffee Table Group Leader(s) ([source](https://github.com/Virtual-Coffee/VC-Contributors/blob/ad1eeb933d68f37cc47647c3f28aa75cd61753f0/coffee-table-groups/leader-docs.md)), Coffee Table Group Leader(s) ([source](https://github.com/Virtual-Coffee/VC-Contributors/blob/ad1eeb933d68f37cc47647c3f28aa75cd61753f0/docs/roles-and-responsibilities/team-leads.md))
- Coffee Table Groups Lead ([source](https://github.com/Virtual-Coffee/VC-Contributors/blob/ad1eeb933d68f37cc47647c3f28aa75cd61753f0/coffee-table-groups/leader-docs.md)), Coffee Table Groups Lead ([source](https://github.com/Virtual-Coffee/VC-Contributors/blob/ad1eeb933d68f37cc47647c3f28aa75cd61753f0/docs/roles-and-responsibilities/team-leads.md))

### Coffees

- Host
- MC
- Room Leader, Breakout Room Leader ([source](https://github.com/Virtual-Coffee/VC-Contributors/tree/ad1eeb933d68f37cc47647c3f28aa75cd61753f0/coffees/RoomLeaders-ConversationFacilitators))
- Note Taker, Notetaker

([source](https://github.com/Virtual-Coffee/VC-Contributors/blob/ad1eeb933d68f37cc47647c3f28aa75cd61753f0/coffees/README.md))

### Lightning Talks

- Speakers
- Mentors
- AV Team (sound and video checks)
- Speaker Coordinators
- Logistics Coordinator
- Host
- Dan

([source](https://github.com/Virtual-Coffee/VC-Contributors/blob/ad1eeb933d68f37cc47647c3f28aa75cd61753f0/lightning-talks/logistics.md))

### Lunch & Learns

- Speaker
- Data Wrangler (DW)
- Logistics Coordinator (LC)
- Marketing
- Moderator
- Host
- Post-production Team:
  - Editor(s)
  - Graphics Coordinator (GC)

([source](https://github.com/Virtual-Coffee/VC-Contributors/blob/ad1eeb933d68f37cc47647c3f28aa75cd61753f0/lunch-and-learns/process-docs.md))

### Monthly Challenges

- Monthly Challenge Facilitators, Monthly Challenge Team
- Monthly Challenge Team Leads

([source](https://github.com/Virtual-Coffee/VC-Contributors/blob/ad1eeb933d68f37cc47647c3f28aa75cd61753f0/monthly-challenges/facilitators-docs.md))

### Other

- Organizational Maintainers, Community Maintainers, Maintainers ([source](https://github.com/Virtual-Coffee/VC-Contributors/blob/ad1eeb933d68f37cc47647c3f28aa75cd61753f0/docs/roles-and-responsibilities/community-maintainers.md))
- Team Leads
- AV Team, AV Team Lead
- Monthly Challenge
- Lightning Talks
- Documentation Team Lead

([source](https://github.com/Virtual-Coffee/VC-Contributors/blob/ad1eeb933d68f37cc47647c3f28aa75cd61753f0/docs/roles-and-responsibilities/team-leads.md))

**Comment by BekahHW** [comment]
Here are my thoughts on the terms:

- **Lead**: someone who takes primary responsibility for a specific project or initiative. They may oversee the project's progress, ensure deadlines are met, and ensure that the project aligns with the VC goals. Leads may also be responsible for rallying and managing a team of volunteers who work on the project.
- **Leader**- responsible for nurturing a welcoming and engaging environment within their specific group, which could focus on various topics such as specific technologies, programming languages, or professional development areas. They might organize regular meetings, lead discussions, ensure active participation, and may also handle conflict resolution within the group. Leaders create spaces that allow members feel connected and supported and make decisions with the best interest of the group in mind.
- **Coordinator**- helps  organize and streamline activities across a specific project area (ie l&ls and coffee table groups). This might involve scheduling, communicating with other volunteers to ensure everyone is informed and on task, handling logistics. Coordinators ensure that the different parts of a project or event are on task and have the adequate support they need.

**Comment by meg-gutshall** [comment]
I'm interested to hear more thoughts, especially from those in the original discussion. I will add my own this week.
cc: @abbeyperini @tkshill @dominicduffin1 @adiati98

**Comment by dominicduffin1** [comment]
Although we are of course not a company, I think its helpful to use an analogy of a company, but recognize that the terminology appropriate to us is not always the same.

So, the Maintainers are like the C-suite, the people who have strategic responsibility for the community and are the ultimate source of authority. Within this, we have Organization Maintainers, who are like the CEO and Treasurer and have legal and financial responsibility for the entire organization, and Community Maintainers, who are like the rest of the C-suite and are more focused on community leadership may have a strategic focus on a specific aspect of the organization.

Then, there is a series of departments, such as Documentation, Coffee Table Groups, Audio-Visual, Monthly Challenge etc. Since we are like a small or medium-sized company, each department is basically a team. The team has a Team Lead (or co-leads) who has overall responsibility and day-to-day decision making responsibility for the team's area of focus, within the strategic parameters set by the Maintainers. Where there is a need or desire to bring other volunteers on board within the team's area of focus, the Team Lead is responsible for coordinating their contributions as well.

Some departments (teams) may be under the direct leadership of a Maintainer or the Maintainers. I believe the Coffees department is under the direct leadership of the Community Maintainers, for example.

Certain teams will have a set of clearly defined volunteer roles under the direction of the Team Lead, such as Host/MC/RL/NT for Coffees. Others may just have a team who work together across the department.

Coffee Table Groups are a bit different, with each Group being a bit like a franchise (albeit an internal one) with the Coffee Table Group Leader (maybe this is somewhere we could use a new term, Owner perhaps?) having full authority over the Group's internal structure, so long as it conforms with the overall rules and ethos of the community (such as the Code of Conduct). The Coffee Table Group department is responsible for coordinating the Groups but not for anything internal to a specific Group.

Beyond this framework, some people may also do other things that are useful to the community but are not part of a specific department. These roles would be outside of the core framework. But, fitting as much as possible into the core structure is probably going to be helpful to best understand, coordinate and support the volunteers and wider community in the long term.

Please feel free to point out any misconceptions in my thinking!

**Reply by tkshill** [reply]
I think @dominicduffin1 's breakdown is a good approximation. Going off @BekahHW 's categories, I think I still feel like the lines between a Lead and a Coordinator are blurred. As in, both can be described, and I know what I would call every volunteer, but I don't think they're completely delineated; there's still overlap. I don't think that matters too much right now though.

I think the `Lead` is ultimately our fundamental volunteer unit. Every _thing_ we do needs a Lead(s). When complexity in a certain area grows, we may need to create a `Coordinator` to support the `Leads`. With the `Maintainers` finally as essentially Coordinators and Stewards for larger scopes of the community.

So like, I can say Coffees have a Tuesday Lead and a Thursday Lead at the moment, and I can see that there could be a Coffee Coordinator role if the organization of those duties ever get separated from active maintainers, and that feels fine.

'Leaders' having a different meaning than just, 'a group of Leads' I think might be tricky down the line, but as of right now, it *should* be fine.

So to Meg's original collection of names in the space:
Our Current Roles and Names are;

- The Maintainers
- The Coordinators
- The Leads

Individual initiatives may make Project specific roles as they need to, and frequently, Leads may appoint Leaders

**Comment by tkshill** [comment]
I think @dominicduffin1 's breakdown is a good approximation. Going off @BekahHW 's categories, I think I still feel like the lines between a Lead and a Coordinator are blurred. As in, both can be described, and I know what I would call every volunteer, but I don't think they're completely delineated; there's still overlap. I don't think that matters too much right now though.

I think the `Lead` is ultimately our fundamental volunteer unit. Every _thing_ we do needs a Lead(s). When complexity in a certain area grows, we may need to create a `Coordinator` to support the `Leads`. With the `Maintainers` finally as essentially Coordinators and Stewards for larger scopes of the community.

So like, I can say Coffees have a Tuesday Lead and a Thursday Lead at the moment, and I can see that there could be a Coffee Coordinator role if the organization of those duties ever get separated from active maintainers, and that feels fine.

'Leaders' having a different meaning than just, 'a group of Leads' I think might be tricky down the line, but as of right now, it *should* be fine.

So to Meg's original collection of names in the space:
Our Current Roles and Names are;

- The Maintainers
- The Coordinators
- The Leads

Individual initiatives may make Project specific roles as they need to, and frequently, Leads may appoint Leaders

**Comment by meg-gutshall** [comment]
Thank you, everyone, for your thoughtful responses! I'm frustrated that GitHub didn't alert me about any replies on this post! 😠

I'm aligned with Kirk's suggestion of sticking to those three roles – Maintainer, Coordinator, and Lead – whenever possible, then letting sub-roles be initiative-specific.

@dominicduffin1, your analogy clicked with something in my brain -- I like it a lot! I do agree that naming things gets a little sticky when it comes to Coffee Table Groups, but it's just that the hierarchy is a bit different. If we go with your franchise analogy, I would say that the Events Maintainer is the Coffee Table Groups owner. When someone wants to start a new franchise, they come to me for approval. After getting the go-ahead, the Coffee Table Groups Coordinator is the person from corporate that helps them get set up and trains them on corporate policy (this is still WIP, but we'll have an issue form shortly and training docs following). Then the new Coffee Table Group Lead is the franchise owner who hires managers (Hosts) to help out. Meh... that's the best I can do right now. Ha ha!

I also agree with @BekahHW's definitions of Lead and Coordinator. I would add a distinction though in that a Coordinator is an administrative role, whereas a Lead drives the initiative forward through hands-on action combined with administrative effort.

To use Dominic's company analogy, a Lead is a one-person consultancy. They develop a SaaS product and win a client. Now they're working with this client's IT team to integrate the product into the client's system, giving support, setting up check-ins, creating deadlines for milestone they need to reach. The Lead wins a second client and decides to hire a virtual assistant (Coordinator). This time around, the Coordinator is doing all the scheduling and facilitating communication for the Lead, however, the Lead still needs to do _some_ administrative work (emails, calendar updates, etc.) to collaborate with the Coordinator or else the Coordinator won't be able to effectively do their job.

I see a Coordinator role as great for someone who may want to do some asynchronous volunteer work. I suppose a Lead can be mainly asynchronous too if the timeline of the initiative isn't pressing, otherwise it may be difficult when it comes to urgency in communication.

**Reply by dominicduffin1** [reply]
I like @meg-gutshall 's expansions of my analogies!

**Comment by meg-gutshall** [comment]
I am interested to hear what @paceaux has to say about this. He and I spoke a few months back about word choices in documentation and he said I could tag him on this matter.

**Comment by meg-gutshall** [comment]
Hey @paceaux! I don't think you received my tag the first time around because the repo wasn't open-source at the time. Do you have any thoughts on the discussion above? I'm interested to hear what you have to say!
